app.js

```diff
@@ -13,22 +13,31 @@ var config = {
 
 require('./index.js').connect(function (context) {
 
+    context.fs.readdir(context.path.join(__dirname, 'api', 'models'), (err, files) => {
+        //Load ALL modules.
+        //Don't get confused by weird logging order! console.log is async and will mix up the actual loading order.
+        files.forEach(file => {
```

Comment:
Whoops, my bad! But as already mentioned, sequelize.sync defines its own order of the table creation so I don't think it makes any difference in which order we load them... But since it does not make a difference, we can leave the code like that :P